### PR TITLE
Update user guide

### PR DESCRIPTION
**Pull request by gohyongjing**
### Changes
* Add more introductory paragraphs to be more "friendly"
  * Describe what are `commands`, `prefixes` and `fields`
  * Describe how to use `cd` in quickstart
* Move `Advanced Info` section to above since reviewers mentioned these information are critical
* Add more terms to glossary
* Change `nr`, `vnr` and `enr` to `ic`, `vic` and `eic` respectively
* Add edit_pair command as COMING SOON

### CODE REVIEW

**Review by heeeyi — APPROVED** [review]
Looks good to merge! Nice work 👍

File: docs/UserGuide.md [hunk 1]
```diff
@@ -9,102 +9,197 @@ title: User Guide
...
+    * Items with `[ellipsis]…` after them can be used 0 or more times.<br>
+      e.g. `add_elderly n/NAME [t/TAG]…` can be used as `add_elderly n/John Doe` (no tags included), `add_elderly n/John Doe t/friend` (1 tag), `add_elderly n/John Doe t/friend t/family` (2 tags) and more.
 
-These are some icons you may see throughout our user guide.
+#### Information Boxes
 
-### Information Box
+You may see the following icons throughout our user guide, which are styled differently according to their purpose.
 
 <div markdown="span" class="alert alert-info">:information_source: **Info:**
```
Comment (Zxun2) [inline docs/UserGuide.md:120]:
```suggestion
<div markdown="span" class="alert alert-info">:information_source: **Info**
```

File: docs/UserGuide.md [hunk 2]
```diff
@@ -9,102 +9,197 @@ title: User Guide
...
 This provides some additional information that you are recommended to know.
 </div>
 
-### Tip Box
+##### Tip Box
 
 <div markdown="block" class="alert alert-primary">:bulb: **Tip:**
```
Comment (Zxun2) [inline docs/UserGuide.md:126]:
```suggestion
<div markdown="block" class="alert alert-primary">:bulb: **Tip**
```

File: docs/UserGuide.md [hunk 3]
```diff
@@ -9,102 +9,197 @@ title: User Guide
...
-
-2. Download the latest `friendlyLink.jar` from [here](https://github.com/AY2223S2-CS2103T-W12-1/tp/releases).
+1. Ensure you have [Java 11](https://www.oracle.com/sg/java/technologies/javase/jdk11-archive-downloads.html) installed in your Computer.
 
-3. Copy the file to an empty folder you want to use as the _home folder_ for your FriendlyLink.
+1. Download the latest `friendlylink.jar` from [here](https://github.com/AY2223S2-CS2103T-W12-1/tp/releases).
 
-4. Open a command terminal, `cd` into the folder you put the jar file in, and use the `java -jar friendlylink.jar` command to run the application.
-   The GUI should appear in a few seconds. <br>
+1. Move the `friendlylink.jar` to an empty folder where you want FriendlyLink to store information.
```
Comment (Zxun2) [inline docs/UserGuide.md:169]:
```suggestion
2. Move the `friendlylink.jar` to an empty folder where you want FriendlyLink to store information.
```

File: docs/UserGuide.md [hunk 4]
```diff
@@ -9,102 +9,197 @@ title: User Guide
...
-    * `delete_elderly S0312312A` : Deletes the elderly with NRIC `S0312312A`.
+    * `add_elderly n/John Doe eic/S1234567A bd/1999-09-09` : Adds an elderly named `John Doe`with NRIC `S1234567A` to FriendlyLink.
 
-    * `clear` : Deletes all data.
+    * `delete_elderly S1234567A` : Deletes the elderly with NRIC `S1234567A`.
 
     * `exit` : Exits the app.
 
-6. Refer to the [Features](#features) below for details of each command.
+1. Refer to the [Features](#features) below for details of each command.
```
Comment (Zxun2) [inline docs/UserGuide.md:190]:
```suggestion
5. Refer to the [Features](#features) below for details of each command.
```

File: docs/UserGuide.md [hunk 5]
```diff
@@ -133,22 +228,22 @@ An elderly can have any number of tags and available dates.
...
-* `add_elderly n/John p/98765432 e/johnd@example.com a/John street enr/S1234567C bd/1950-02-03 re/NORTH r/HIGH t/lonely dr/2023-06-03,2023-06-17`
+* `add_elderly n/John Wick ic/S1234567C bd/1950-02-03`
+* `add_elderly n/Betsy Crowe p/98765432 e/johnd@example.com a/John street eic/S1234567C bd/1950-02-03 re/NORTH r/HIGH t/lonely dr/2023-06-03,2023-06-17`
 
 #### Adding a volunteer: `add_volunteer`
 
 Adds a volunteer to FriendlyLink.
 
-Format: `add_volunteer vnr/NRIC n/NAME [p/PHONE] [e/EMAIL] [a/ADDRESS] [bd/BIRTH_DATE] [re/REGION] [t/TAG]…​ [mt/MEDICAL_QUALIFICATIONS]… [dr/START_DATE, END_DATE]…​`
+Format: `add_volunteer vic/NRIC n/NAME bd/BIRTH_DATE [p/PHONE] [e/EMAIL] [a/ADDRESS] [re/REGION] [t/TAG]…​ [mt/MEDICAL_QUALIFICATIONS]… [dr/START_DATE, END_DATE]…​`
```
Comment (Zxun2) [inline docs/UserGuide.md:238]:
```suggestion
Format: `add_volunteer vic/NRIC n/NAME bd/BIRTH_DATE [p/PHONE] [e/EMAIL] [a/ADDRESS] [re/REGION] [t/TAG]…​ [mt/MEDICAL_QUALIFICATIONS]… [dr/AVAILABLE_DATE_START, AVAILABLE_DATE_END]…​`
```

File: docs/UserGuide.md [hunk 6]
```diff
@@ -133,22 +228,22 @@ An elderly can have any number of tags and available dates.
...
 * Every volunteer must have a unique `NRIC`.
 * Alphabets in `NRIC` are case-insensitive.
 * The `REGION` can only take 5 values: `NORTH`, `NORTHEAST`, `CENTRAL`, `WEST` or `EAST`.
 * Dates specified should follow the format `YYYY-MM-DD`. 
 * For available dates, the start date should be before the end date.
 * Phone number specified can only be numeric characters, and must be at least 3 digits long.
-* The `MEDICAL_QUALIFICATION` takes the form `SKILL_NAME LEVEL`. The `LEVEL` can only take 3 values: `BASIC`
-  , `INTERMEDIATE` or `HIGH`. Exmaple: `CPR BASIC`, `AED INTERMEDIATE`.
+* The `MEDICAL_QUALIFICATION` takes the form `SKILL_NAME LEVEL`. The `LEVEL` can only take 3 values: `BASIC`, `INTERMEDIATE` or `HIGH`. Example: `CPR BASIC`, `AED INTERMEDIATE`.
 * `START_DATE, END_DATE` represents the start and end of the dates that the volunteer is available.
```
Comment (Zxun2) [inline docs/UserGuide.md:247]:
```suggestion
* `START_DATE, END_DATE` represents the start and end of the dates that the volunteer is available.
```
Same here

File: docs/UserGuide.md [hunk 7]
```diff
@@ -274,8 +366,25 @@ Format: `edit NRIC [n/NAME] [nr/NRIC] [p/PHONE] [e/EMAIL] [a/ADDRESS] [bd/BIRTH_
 Examples:
 
 * `edit S2233556T p/91642345 re/NORTH` Edits the phone number of the person identified by `S2233556T` to be `91642345` and region to be `NORTH`.
-* `edit S8833657U ag/68 r/HIGH` Edits the age of the person identified by `S8833657U` to be `68` and risk level to be `HIGH`. However, if `S8833657U` identifies a volunteer, the risk level edit will be ignored.
+* `edit S8833657U re/CENTRAL r/HIGH` Edits the region of the person identified by `S8833657U` to be `CENTRAL` and risk level to be `HIGH`. However, if `S8833657U` identifies a volunteer, the risk level edit will be ignored.
+
+#### Editing a pair by index: `edit_pair` (COMING SOON)
+
+<div markdown="span" class="alert alert-info">:information_source: **Info:**
```
Comment (Zxun2) [inline docs/UserGuide.md:373]:
```suggestion
<div markdown="span" class="alert alert-info">:information_source: **Info**
```

File: docs/UserGuide.md [hunk 8]
```diff
@@ -274,8 +366,25 @@ Format: `edit NRIC [n/NAME] [nr/NRIC] [p/PHONE] [e/EMAIL] [a/ADDRESS] [bd/BIRTH_
...
+This feature has not been implemented yet.
+</div>
+
+Edits an existing pair based on their index in the pairs list.
+
+Format: `edit_pair INDEX [eic/ELDERLY_NRIC] [vic/VOLUNTEER_NRIC]`
+
+* Edits the pair at the specified `INDEX`. The index refers to the index number shown in the displayed pairs list. The index **must be a positive integer** 1, 2, 3, …​
```
Comment (Zxun2) [inline docs/UserGuide.md:381]:
```suggestion
* Edits the pair at the specified `INDEX`. 
```
I think can put `INDEX` in the advanced info so no need to repeat it multiple times

**Review by Zxun2 — APPROVED** [review]
Looks good ✅

File: docs/UserGuide.md [hunk 9]
```diff
@@ -9,102 +9,197 @@ title: User Guide
...
-![Ui](images/Ui.png)
+1. Double-click on the `friendlylink.jar` file. If the app does not open, follow the following steps instead.
+    * Open a terminal
+      * On **Windows**: Click Start and search for `Command Prompt`
+      * On **macOS**: Open Launchpad and search for `terminal`
+    * Move into the folder you are keeping FriendlyLink by entering `cd FILE/PATH/TO/FRIENDLYLINK` into the terminal (without the angled brackets)
+    * Open the app by entering `java -jar friendlylink.jar` into the terminal.
+   
+   The GUI should appear in a few seconds.
+   ![Ui](images/Ui.png)
```
[the remark this answers is not in the document]
Comment (gohyongjing) [reply docs/UserGuide.md:179]: =)

File: docs/UserGuide.md (hunk 3 above)
Comment (gohyongjing) [reply docs/UserGuide.md:169]: I think we can use github auto numbering for this, so we will leave all as 1.

File: docs/UserGuide.md [hunk 10]
```diff
@@ -9,102 +9,197 @@ title: User Guide
...
 
-3. Copy the file to an empty folder you want to use as the _home folder_ for your FriendlyLink.
+1. Download the latest `friendlylink.jar` from [here](https://github.com/AY2223S2-CS2103T-W12-1/tp/releases).
 
-4. Open a command terminal, `cd` into the folder you put the jar file in, and use the `java -jar friendlylink.jar` command to run the application.
-   The GUI should appear in a few seconds. <br>
+1. Move the `friendlylink.jar` to an empty folder where you want FriendlyLink to store information.
 
-![Ui](images/Ui.png)
+1. Double-click on the `friendlylink.jar` file. If the app does not open, follow the following steps instead.
```
[the remark this answers is not in the document]
Comment (gohyongjing) [reply docs/UserGuide.md:171]: Same as above

File: docs/UserGuide.md (hunk 8 above)
Comment (gohyongjing) [reply docs/UserGuide.md]: Resolved in #[4e9d019](https://github.com/AY2223S2-CS2103T-W12-1/tp/pull/181/commits/4e9d019b894de4f64d9750e12ad8f1b1a16fed11)

**Pull request merged**